types/choose_sequence/example.json5

```diff
@@ -0,0 +1,51 @@
+{
+  type: "choose_sequence",
+  lang: "en",
+  tags: ["development", "javascript"],
+  card: {
+    question: "Turn the arguments of a method into a array:\
+      ```function myArray() {\
+          var a = ???;\
+          a = Array.prototype.???.call(a);\
+          return a;\
+        }```",
+    answers: [
+      {
+        text: "arguments",
+        sequenceNumber: 0
```

Comment:
we can follow the same agreement as enki - or that's too complicated?

Reply:
You mean that for example first 2 options here will be correct answers for the question?

Reply:
not sure if that's clear approach - just reminded about such case